File: Codechef/Attendu/Attendu.py
T = int(input())
while T:
    T -= 1
    N = int(input())
    B = input()
    c1 = 0
    for i in B:
        if i == "1":
            c1 += 1
    remain = 120 - N + c1
    atten_perc = (remain/120) * 100
    if atten_perc >= 75:
        print("YES")
    else:
        print("NO")

File: Codechef/Attendu/Attendu.cpp
#include <iostream>
using namespace std;

int main()
{
    int T;
    cin >> T;
    while (T--)
    {
        int N, remain;
        float atten_perc;
        string B;
        cin >> N;
        cin >> B;
        remain = 120 - N;
        atten_perc = (remain/120) * 100;
        if (atten_perc >= 75)
        {
            cout << "YES" << endl;
        }
        else
        {
            cout << "NO" << endl;
        }

    }



    return 0;
}